### DISCUSSION TITLE
Table and THead issue

### DISCUSSION THREAD

**Discussion by LechK** [twilio-labs/paste#2881] [Ideas] [answered]
Hi, I have noticed, that if I give for `Table` `scrollHorizontally` property, my child `THead`'s `stickyHeader` property does not work, TableHead is not sticking anymore. Is it proper behavior or it could be related to bug?

**Comment by nkrantz** [comment, marked as answer]
Hi @LechK! Scrolling sticky header isn't a common pattern that we've seen. Could you give us some more information about your use case so that we can determine whether to investigate this further? What are you trying to accomplish with your table?

**Reply by LechK** [reply]
Hi @nkrantz! Thank You for your answer. Could You please share Paste components sandbox, so I could reproduce that? I could not find it

**Reply by nkrantz** [reply]
[Here's a starter CodeSandbox with Paste ready to go](https://codesandbox.io/s/paste-starter-template-rz6u3), but no need to make a repro for this. I've checked it out on my end and confirmed that our current implementation doesn't allow for both properties at the same time. 

Before we decide to work on an implementation that allows both `scrollHorizontally` and `stickyHeader`, could you give us some more information about your needs? If we know what your use case is for a sticky/scrolling table, it'll help us to prioritize fixing the bug. Thanks!

**Reply by serifluous** [reply]
Hey @LechK ! We'll mark this closed for now due to inactivity, but can definitely open it back up again if you're able to share more info about your use case. Thanks!
